Obsidian/Commands/MainCommandModule.cs

```diff
@@ -493,4 +493,57 @@ private ChatMessage SendCommandUsage(string commandUsage)
         commands.AddExtra(usage);
         return commands;
     }
+
+    [Command("give")]
+    [CommandInfo("Give an item for a specific player", "/give <player> <block> [<amount>]")]
+    public async Task GiveAsync(CommandContext ctx, string player, string block) => await GiveAsync(ctx, player, block, 1);
+
+    [CommandOverload]
+    public async Task GiveAsync(CommandContext ctx, string player, string block, short amount)
+    {
+        try
+        {
+            IReadOnlyList<Player> players = new List<Player>();
+            if(player.EqualsIgnoreCase("@a")) players = ctx.Server.Players.Select(p=>p as Player).ToList();
+            if(!player.StartsWith("@") /*&& player.IsValidUsername()*/) players = new List<Player> { ctx.Server.GetPlayer(player, StringComparison.OrdinalIgnoreCase) as Player };
```

Comment:
This looks weird. A better approach probably is `if (player[0] == '@')` and then do further checks inside this one.

Reply:
Started fixing in 128ee2200bf3a410ea5502e1f57b86f7cc1eb885.